README.md

```diff
@@ -12,32 +12,40 @@ Lead Maintainer: [Wyatt Preul](https://github.com/geek)
 
 ### Basic
 ```javascript
-var Wreck = require('wreck');
+const Wreck = require('wreck');
 
-Wreck.get('https://google.com/', function (err, res, payload) {
+Wreck.get('https://google.com/', (err, res, payload) => {
+    /* do stuff */
+});
+```
+
+```javascript
+const Wreck = require('wreck');
+
+Wreck.post('https://posttestserver.com/post.php', { payload: { hello: 'post' } }, (err, res, payload) => {
     /* do stuff */
 });
 ```
 
 ### Advanced
 ```javascript
-var Wreck = require('wreck');
+const Wreck = require('wreck');
 
-var method = 'GET'; // GET, POST, PUT, DELETE
-var uri    = '/';
-var readableStream = Wreck.toReadableStream('foo=bar');
+const method = 'GET'; // GET, POST, PUT, DELETE
+const uri    = '/';
```

Comment:
While you're here, it would be a good time to remove the equal sign alignment :-)